test/scenarios/filters/object_filters.jt

```diff
@@ -4,5 +4,5 @@
 ]
 {.a[].length > 1}
 {3 in .a}.{.a[].includes(4)}
-{typeof .b === "number"}
+{!(typeof .b === "string")}
```

Comment:
Let's update this test case with 
[
  {a: [1, 2], b: "1"}, {a: [3, 4], b: "2"}, {a: [5, 3], b: 3}, {a:[3], b: 4}, {b: 5}, {a: [3, 4], b: 1234}
]

Reply:
done, made small modification so that I don't need to change the outputs